### ISSUE TITLE
v1.0.4 Has a Broken Path

### ISSUE THREAD

**Issue by jmuldoon** [spf13/pflag#218]
edit: updated how to recreate

`go get -u github.com/golangci/golangci-lint/cmd/golangci-lint` will fail when hitting this dep

```
go get: github.com/spf13/pflag@v1.0.3 updating to
        github.com/spf13/pflag@v1.0.4: parsing go.mod:
        module declares its path as: github.com/spf13/pflags
                but was required as: github.com/spf13/pflag
```

**Comment by cfromknecht** [comment]
we are seeing the same issue here https://travis-ci.org/lightningnetwork/lnd/jobs/586171113

**Comment by cfromknecht** [comment]
It seems the issue is being caused because `v1.0.4` is still requiring `v1.0.3`, which had the typo
```
require github.com/spf13/pflag v1.0.3
```

**Comment by mback2k** [comment]
Go 1.13 `go get -u` also reports this:

```
verifying github.com/spf13/pflag@v1.0.4/go.mod: checksum mismatch
	downloaded: h1:5F7vF+0t+QPgqofPFTZZG2xnxtA/FGYmPu6WbdsXIvo=
	sum.golang.org: h1:SpGzs/otoxJg1kCwgj9MIIWODxcr65fui/Dn/ZF6H9E=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

**Comment by aelse** [comment]
I encountered the same. Please tag v1.0.5 after merge as 1.0.4 is poisoned. Thank you dear maintainers 😄

**Comment by rmilejcz** [comment]
is there any fix for this or are we just waiting on lib maintainers?

**Comment by therealmitchconnors** [comment]
#219 merged.  Creating rc release now, to ensure we have this fixed.

**Comment by therealmitchconnors** [comment]
v1.0.5-rc1 is out https://github.com/spf13/pflag/releases/tag/v1.0.5-rc1

Please report here if this resolves your issue.

**Comment by cfromknecht** [comment]
thank you @therealmitchconnors! 🎉

**Comment by therealmitchconnors** [comment]
@cfromknecht does that mean it works for you?  It seems to work for me, but I want to be sure before I create the release...

**Comment by cfromknecht** [comment]
@therealmitchconnors yes `v1.0.5-rc1` is working for me, thanks again!

**Comment by therealmitchconnors** [comment]
ok, based on the feedback, I have shipped v1.0.5.  Please re-run `go get -u` to resolve this message.
